admin/lib/solidus_admin/engine.rb

```diff
@@ -40,6 +40,7 @@ class Engine < ::Rails::Engine
 
     initializer "solidus_admin.importmap.assets" do |app|
       app.config.assets.paths << SolidusAdmin::Engine.root.join("app/javascript")
+      app.config.assets.paths << SolidusAdmin::Engine.root.join("app/components")
```

Comment:
Neat: thoughts about using `push` with the two arguments to avoid referencing twice `app.config.assets.paths`?